x/concentrated-liquidity/lp_test.go

```diff
@@ -31,7 +31,22 @@ func (s *KeeperTestSuite) TestMint() {
 
 	asset0, asset1, err := s.App.ConcentratedLiquidityKeeper.Mint(s.Ctx, poolId, s.TestAccs[0], liquidity, lowerTick, upperTick)
 	s.Require().NoError(err)
-
 	s.Require().Equal(sdk.NewInt(1), asset0)
 	s.Require().Equal(sdk.NewInt(5000), asset1)
+
+	// check position state
+	position, err := s.App.ConcentratedLiquidityKeeper.GetPosition(s.Ctx, poolId, s.TestAccs[0], lowerTick, upperTick)
+	s.Require().NoError(err)
+	s.Require().Equal(sdk.NewInt(1517), position.Liquidity)
+
+	// check tick state
+	lowerTickInfo, err := s.App.ConcentratedLiquidityKeeper.GetTickInfo(s.Ctx, poolId, lowerTick)
+	s.Require().NoError(err)
+	s.Require().Equal(sdk.NewInt(1517), lowerTickInfo.LiquidityGross)
+	s.Require().Equal(sdk.NewInt(1517), lowerTickInfo.LiquidityNet)
+
+	upperTickInfo, err := s.App.ConcentratedLiquidityKeeper.GetTickInfo(s.Ctx, poolId, upperTick)
+	s.Require().NoError(err)
+	s.Require().Equal(sdk.NewInt(1517), upperTickInfo.LiquidityGross)
+	s.Require().Equal(sdk.NewInt(-1517), upperTickInfo.LiquidityNet)
```

Comment:
can you explain the math behind getting 1517?

Reply:
It's the liquidity we put in initially as the testing params

Reply:
Should we add a comment for that?

Reply:
added